small adjustments

GregoryKucherkov opened the pull request:
- adjust instructions for user
- add ASCII Darth Vader logo
- adjust func handle_all_birthdays to print out list of birthdays for <parametr> days

ArthurMykhailiuk commented on models.py:
```diff
@@ -103,13 +103,13 @@ def find(self, name):
             del self.data[name]
         else:
-            raise KeyError("Name not found")
+            raise KeyError("Name not found young Jedi. Enter <?> to faind out all comands")
```
Change faind to find

ArthurMykhailiuk reviewed (commented):
Look’s good

GregoryKucherkov committed:
small adjustments

- adjust instructions for user
- add ASCII Darth Vader logo
- adjust func handle_all_birthdays to print out list of birthdays for
  <parametr> days

LilianaLukash committed:
Merge branch 'main' into style/birthdays-list
